app/views/curation_concerns/vector_works/_related_vector_files.html.erb

```diff
@@ -14,11 +14,11 @@
       </tr>
     </thead>
     <tbody>
-      <%= render presenter.vector_file_presenters %>
+      <%= render @presenter.vector_file_presenters %>
```

Comment:
You should be able to do:

```
<%= render partial: 'member', collection: @presenter.vector_file_presenters %>
```

And then you won't need `app/views/curation_concerns/file_sets/_file_set.html.erb`